scripts/exglobal_snow_analysis.py

```diff
@@ -18,9 +18,20 @@
     config = cast_strdict_as_dtypedict(os.environ)
 
     # Instantiate the snow analysis task
-    anl = SnowAnalysis(config)
-    if anl.task_config.cyc == 0:
-        anl.prepare_IMS()
-    anl.initialize()
-    anl.execute()
-    anl.finalize()
+    SnowAnl = SnowAnalysis(config)
```

Comment:
Variable names should be in snake_case. CamelCase is reserved for class names. See [PEP-8 section on naming conventions](https://peps.python.org/pep-0008/#prescriptive-naming-conventions).

```suggestion
    snow_anl = SnowAnalysis(config)
```

Reply:
ok, just noting here that we will need to modify others in the future to be consistent but I'm happy to go with snake case here